feeds/pl.json

```diff
@@ -120,9 +120,11 @@
         },
         {
             "name": "Kielce",
-            "type": "transitland-atlas",
-            "transitland-atlas-id": "f-u3n9-kielce",
-            "fix": true
+            "type": "http",
+            "url": "https://cdn.zbiorkom.live/gtfs/kielce.zip",
+            "license":{
+                "spdx-identifier": "MIT"
```

Comment:
The test looks like this one has some broken references as well, like the previous one.

Please keep fix: true

Reply:
Added.